tools/code-owners-parser/CodeOwnersParser/FileHelpers.cs

```diff
@@ -22,10 +23,42 @@ public static string GetFileOrUrlContents(string fileOrUrl)
 
         private static string GetUrlContents(string url)
         {
+            int maxRetries = 3;
+            int attempts = 1;
+            int delayTimeInMs = 1000;
             using HttpClient client = new HttpClient();
-            HttpResponseMessage response =
-                client.GetAsync(url).ConfigureAwait(false).GetAwaiter().GetResult();
-            return response.Content.ReadAsStringAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+            while (attempts <= maxRetries)
```

Comment:
Might not be necessary in this case but there are some more standard retry patterns for HttpClient, see https://stackoverflow.com/questions/19260060/retrying-httpclient-unsuccessful-requests for examples.